test/basic-querying.test.js

```diff
@@ -573,59 +573,66 @@ describe('basic-querying', function() {
         });
     });
 
-    var itWhenIlikeSupported = connectorCapabilities.ilike ? it : it.skip.bind(it);
-
-    itWhenIlikeSupported('should support "like" that is satisfied', function(done) {
-      User.find({where: {name: {like: 'John'}}}, function(err, users) {
-        if (err) return done(err);
-        users.length.should.equal(1);
-        users[0].name.should.equal('John Lennon');
-        done();
-      });
-    });
-
-    itWhenIlikeSupported('should support "like" that is not satisfied', function(done) {
-      User.find({where: {name: {like: 'Bob'}}}, function(err, users) {
-        if (err) return done(err);
-        users.length.should.equal(0);
-        done();
-      });
-    });
-
-    var itWhenNilikeSupported = connectorCapabilities.nilike ? it : it.skip.bind(it);
+    var itWhenIlikeSupported = connectorCapabilities.ilike !== false;
+    bdd.itIf(itWhenIlikeSupported, 'ilike', function() {
```

Comment:
Will this run both the `like` and `ilike` tests if `itWhenIlikeSupported` is true?